ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift

```diff
@@ -10,10 +10,15 @@ class ButtonDemoController: DemoController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        view.backgroundColor = Colors.surfaceSecondary
 
         for size in MSFButtonSize.allCases {
             addTitle(text: "\(size.description.capitalized) size")
             for style in MSFButtonStyle.allCases {
+                let floatingStyle = style == .accentFloating || style == .subtleFloating
```

Comment:
nit: If you would like to filter doing it before the loop would be easy to understand.